src/test/ui/error-codes/E0023.stderr

```diff
@@ -5,16 +5,10 @@ LL |     Apple(String, String),
    |     --------------------- tuple variant defined here
 ...
 LL |         Fruit::Apple(a) => {},
-   |         ^^^^^^^^^^^^^^^ expected 2 fields, found 1
-   |
-help: use `_` to explicitly ignore each field
-   |
-LL |         Fruit::Apple(a, _) => {},
-   |                       ^^^
-help: use `..` to ignore the rest of the fields
-   |
-LL |         Fruit::Apple(a, ..) => {},
-   |                       ^^^^
+   |         ^^^^^^^^^^^^^^-
+   |         |             |
+   |         |             help: use `_` to explicitly ignore each field
+   |         expected 2 fields, found 1
```

Comment:
Ideally this wouldn't show the `help` inline. Is there any way to force it to be how it was before?

Reply:
`span_suggestion_verbose` always displays the suggestion in its own subdiagnostic window.

Reply:
Ah, I didn't realize that! The docs just say:

> Always show the suggested change.

I didn't realize that meant "always show in its own window". Do you think you could update the docs? If not, I can perhaps do it.

Reply:
I can look into updating the docs, there's a lot that could be improved there :)